core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -129,50 +218,323 @@ export const border = () => {
+              {
+                '<Text appearance="default" className="border border-right-0 bg-secondary-lightest p-8 d-inline-block mr-6">right-0</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-bottom-0 bg-secondary-lightest p-8 d-inline-block mr-6">bottom-0</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-left-0 bg-secondary-lightest p-8 d-inline-block">left-0</Text>'
+              }
+            </code>
+          </pre>
+        </div>
+        <Text appearance="subtle" className="mb-2 font-size-xs">
+          These utilities let you remove borders from specific sides of an element.
+        </Text>
       </div>
-      &nbsp;
-      <div className="DocPage-codeBlock w-50 pl-10">
-        {'<span className="border bg-secondary-lightest p-8 d-inline-block mr-6">border</span>'}
-        <br />
-        {'<span className="border-top bg-secondary-lightest p-8 d-inline-block mr-6">top</span>'}
-        <br />
-        {'<span className="border-right bg-secondary-lightest p-8 d-inline-block mr-6">right</span>'}
-        <br />
-        {'<span className="border-bottom bg-secondary-lightest p-8 d-inline-block mr-6">bottom</span>'}
-        <br />
-        {'<span className="border-left bg-secondary-lightest p-8 d-inline-block">left</span>'}
+
+      {/* Border radius examples */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Border Radius
+        </Heading>
+        <div className="mb-4">
+          <Text appearance="default" className="border rounded-2-5 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-2-5
+          </Text>
+          <Text appearance="default" className="border rounded-05 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-05
+          </Text>
+          <Text appearance="default" className="border rounded-10 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-10
+          </Text>
+          <Text appearance="default" className="border rounded-15 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-15
+          </Text>
+          <Text appearance="default" className="border rounded-20 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-20
+          </Text>
+          <Text appearance="default" className="border rounded-30 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-30
+          </Text>
+          <Text appearance="default" className="border rounded-40 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-40
+          </Text>
+          <Text appearance="default" className="border rounded-full bg-secondary-lightest p-8 d-inline-block">
+            rounded-full
+          </Text>
```

Comment:
Remove border classes from the Text, use layout or wrapper components instead for better separation of concern